build/scripts/build-impl.xml

```diff
@@ -276,15 +276,15 @@
     <target name="jar" depends="git.details,compile,compile-aspectj"
         description="Create eXist-db unsigned jar files">
         <filter token="version" value="${project.version}"/>
```

Comment:
Not unless RC, SNAPSHOT etc should be part of installer filename and so on like now